.github/stale.yml

```diff
@@ -33,15 +33,30 @@ pulls:
   daysUntilStale: 60
   daysUntilClose: 7
   markComment: >
-    This pull request/issue has been marked as stale due to 60 days of inactivity.
+    This pull request has been marked as stale due to 60 days of inactivity.
     It will be closed in 1 week if no further activity occurs. If you think
-    that’s incorrect or this pull request/issue requires a review, please simply
-    write any comment. If closed, you can revive the PR/issue at any time and @mention
-    a reviewer or discuss it on the dev@druid.apache.org list.
+    that's incorrect or this pull request should instead be reviewed, please simply
+    write any comment. Even if closed, you can still revive the PR at any time or
+    discuss it on the dev@druid.apache.org list.
     Thank you for your contributions.
   unmarkComment: >
     This pull request/issue is no longer marked as stale.
   closeComment: >
     This pull request/issue has been closed due to lack of activity. If you think that
-    is incorrect, or the pull request/issue requires review, you can revive the PR/issue at
+    is incorrect, or the pull request requires review, you can revive the PR at any time.
+
+issues:
+  daysUntilStale: 280
```

Comment:
How about 180? I think 6 months is plenty of time

Reply:
Plenty of time for what? I was thinking it makes sense to go through at least 3 release cycles. Also, it's common to use issues as placeholders for 'stuff to be done later' and 6 months can be short for that, imo.

Reply:
saw the dev@ thread , this rationale is very thoughtful .